core/vm/interface.go

```diff
@@ -24,18 +26,29 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
+type AccountExtraDataStateReader interface {
+	// Return nil for public contract
+	ReadPrivacyMetadata(addr common.Address) (*state.PrivacyMetadata, error)
```

Comment:
MINOR: For consistency with the MinimalApiState I would rename those function `Get...` better than `Read...`

Reply:
changed